app/decorators/service_provider_session.rb

```diff
@@ -118,12 +118,12 @@ def url_options
 
   def request_url_params
     @request_url_params ||= begin
-      if request_url.present?
-        UriService.params(request_url)
-      else
-        {}
-      end
-    end
+                              if request_url.present?
+                                UriService.params(request_url)
+                              else
+                                {}
+                              end
+                            end
```

Comment:
Can we undo this indentation?
```suggestion
      if request_url.present?
        UriService.params(request_url)
      else
        {}
      end
    end
```

Reply:
Fixed.